### ISSUE TITLE
Manifest file and DevServer with publicUrl

### ISSUE THREAD

**Issue by FVesely** [o2ps/WebpackNetteAdapter#15]
config.neon
```json
webpack:
	devServer:
		url: http://webpack-dev-server:8080
		publicUrl: http://localhost:8080
	manifest:
		name: dist/manifest.json
```
WebpackNetteAdapter is trying to load manifest content from http://localhost:8080/dist/manifest.json instead of http://webpack-dev-server:8080/dist/manifest.json

**Comment by jiripudil** [comment]
Yep, that's a bug. Thanks for the report!

**Comment by jiripudil** [comment]
I've released 1.3.1 with the fix 🎉

**Comment by FVesely** [comment]
@jiripudil Thank you, that was fast.
